source/databricks/package/integration_events_persister.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+sys.path.append(r'/workspaces/opengeh-wholesale/source/databricks')
+
+from pyspark.sql import DataFrame
+from pyspark.sql.functions import (
+    year, month, dayofmonth, col)
+
+
+def integration_events_persister(streamingDf: DataFrame, checkpoint_path: str, integration_events_path: str):
```

Comment:
For naming consistency in domain (remember the file name):
```suggestion
def integration_event_persister(streamingDf: DataFrame, checkpoint_path: str, integration_events_path: str):
```

Reply:
Naming consistency with what? IntegrationEventListener? It fine either way for me, I would just like to know why one is better than the other

Reply:
I don't know if one is better than the other. I, however, do encourage consistency.